xgboost_ray/main.py

```diff
@@ -291,6 +275,11 @@ def train(self, rabit_args: List[str], params: Dict[str, Any],
 
         local_params = params.copy()
 
+        if "xgb_model" in kwargs:
+            if isinstance(kwargs["xgb_model"], bytes):
+                # bytearray type gets lost in remote actor call
+                kwargs["xgb_model"] = bytearray(kwargs["xgb_model"])
+
```

Comment:
is this an issue/bug?

Reply:
I think so. I'll check this tomorrow and write a small reproducible script, possibly open an issue on ray core